gems/quilt_rails/dev.yml

```diff
@@ -0,0 +1,12 @@
+name: react-server
+up:
+  - ruby: 2.3.7
```

Comment:
Too old ruby. BTW we need to make sure this gem works with Ruby 2.5+ and Rails 5.2, so we may need more tests in the matrix.

Reply:
it definitely does, `web-rails-proving-ground` is where I initially prototyped this and that repo uses 2.6.x